aviary/interface/methods_for_level2.py

```diff
@@ -404,36 +405,8 @@ def _update_metadata_from_subsystems(self):
                 self.phase_info[phase_name]['external_subsystems'])
             for subsystem in external_subsystems:
                 meta_data = subsystem.meta_data.copy()
-
-                state_info = subsystem.get_states()
```

Comment:
how can we get rid of all this code?

Reply:
I am not even sure why we were originally removing all of these from the meta_data, but we don't seem to need to do it anymore.  Maybe it's because we are adding parameter/etc of the default subsystems now using the same methods as we do for external ones.